client/src/components/MediaCard/MediaCard.css

```diff
@@ -1,9 +1,13 @@
 .MediaCard {
```

Comment:
Is the MediaCard needed for your new cards at all?

Reply:
@erinkelley27  No! Not needed.